msteams-platform/resources/schema/manifest-schema-dev-preview.md

```diff
@@ -1032,8 +1032,33 @@ The `extensions.runtimes` array configures the sets of runtimes and actions that
 |`actions.displayName`| String | 64 characters | | Specifies the display name of the action and it isn't the label of a button or a menu item that invokes the action (which is configured with `tabs.groups.controls.label`).|
 |`actions.pinnable`| Boolean | | | Specifies that a task pane supports pinning, which keeps the task pane open when the user changes the selection. <br>Default value: `false`|
 |`actions.view`| String | 64 characters | | Specifies the view where the page must be opened. It's used only when `actions.type` is `openPage`. |
-|`actions.multiselect`| Boolean | | | Specifies whether the end user can select multiple items, such as multiple email messages, and apply the action to all of them.<br>Default value: `false`|
-|`actions.supportsNoItemContext`| Boolean | | | Allows task pane add-ins to activate without the Reading Pane enabled or a message selected.<br>Default value: `false`|
+|`actions.multiselect`| Boolean | | | Specifies whether the end user can select multiple items, such as multiple email messages, and apply the action to all of them.|
```

Comment:
Hi @alison-mk,
Kindly share the reason for removing the default value for properties `multiselect` and `supportsNoItemContext`.

Reply:
Thanks for catching this! It was unintentional. I'll add the default values back in.